src/components/tab-group/tab/bl-tab.ts

```diff
@@ -142,13 +150,15 @@ export default class BlTab extends LitElement {
         role="tab"
         class="container"
         @click="${() => this.select()}"
+        aria-selected="${this.selected}"
       >
         <div class="title-container">
           <div class="title">${icon} ${title} ${badge}</div>
           ${caption}
         </div>
         ${helpTooltip}
       </button>
+      <span class="divider"></span>
```

Comment:
What was the motivation behing this decision? Did you have issues about putting border with pseude-element and outline?

Reply:
earlier the design demanded to have a padding on outline from left side but not the right side, which was harder to achieve with the pseudo classes, but since we have now changed the design to have even paddings from both the sides, let me try to remove this redundant `<span>` again and check.

Reply:
removed it.